subsys/bluetooth/host/keys.h

```diff
@@ -74,19 +77,108 @@ struct bt_keys {
 #define BT_KEYS_STORAGE_LEN     (sizeof(struct bt_keys) - \
 				 offsetof(struct bt_keys, storage_start))
 
-void bt_keys_foreach(int type, void (*func)(struct bt_keys *keys, void *data),
+/**
+ * @brief Get a call through the callback for each key with the same type
+ *
+ * @param type Key type.
+ * @param func Callback function to be called when a matched record is found.
+ * @param data User data to be passed to the callback function.
+ */
+void bt_keys_foreach_type(int type, void (*func)(struct bt_keys *keys, void *data),
 		     void *data);
```

Comment:
Looks like the indentation/alignment of the split line is now wrong, after the function name got longer